docs/src/introduction/forc_project.md

```diff
@@ -55,36 +55,37 @@ We now compile our project with `forc build`, passing the flag `--print-finalize
 
 ```console
 $ forc build --print-finalized-asm
+...
 .program:
 ji   i4
 noop
 DATA_SECTION_OFFSET[0..32]
 DATA_SECTION_OFFSET[32..64]
 lw   $ds $is 1
 add  $$ds $$ds $is
-lw   $r1 $fp i73              ; load input function selector
-lw   $r0 data_1               ; load fn selector for comparison
-eq   $r0 $r1 $r0              ; function selector comparison
-jnzi $r0 i11                  ; jump to selected function
-rvrt $zero                    ; revert if no selectors matched
-lw   $r0 data_0               ; literal instantiation
-ret  $r0
+lw   $r0 $fp i73              ; load input function selector
+lw   $r1 data_0               ; load fn selector for comparison
+eq   $r2 $r0 $r1              ; function selector comparison
+jnzi $r2 i12                  ; jump to selected function
+movi $$tmp i123               ; special code for mismatched selector
+rvrt $$tmp                    ; revert if no selectors matched
+ret  $one
 .data:
-data_0 .bool 0x01
-data_1 .u32 0x2151bd4b
+data_0 .word 559005003
 
   Compiled contract "my-fuel-project".
-  Bytecode size is 68 bytes.
+  Bytecode size is 60 bytes.
 ```
 
-To test this contract, use `forc test`:
+To test this contract, use `cargo test`:
```

Comment:
I've removed from this line onwards in https://github.com/FuelLabs/sway/pull/2264 - would you mind reverting the unrelated lines that touch the test parts to avoid conflicts?